File: openapi/code_samples/nodejs/access_token/code.js
var request = require('request');
var options = {
  'method': 'POST',
  'url': 'https://auth.reloadly.com/oauth/token',
  'headers': {
    'Content-Type': 'application/json'
  },
  body: JSON.stringify({
    "client_id": "qwcLzXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    "client_secret": "7kscVxQZ32-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    "grant_type": "client_credentials",
    "audience": "https://topups.reloadly.com"
  })

};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});

File: openapi/code_samples/python/access_token/code.py
import requests
import json

url = "https://auth.reloadly.com/oauth/token"

payload = json.dumps({
  "client_id": "qwcLzXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
  "client_secret": "7kscVxQZ32-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
  "grant_type": "client_credentials",
  "audience": "https://topups.reloadly.com"
})
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)